build.html

```diff
@@ -1,46 +1,47 @@
 <!-- HEADER -->
 <div class="fl-viewport-header {{#if location}}fl-viewport-header-left{{/if}}">
   {{#if location}} {{#if canGoBack}}
-  <span class="nav-left fl-menu-back" data-fl-navigate-back>
-        <i class="fa fa-angle-left" aria-hidden="true"></i> <span class="btn-label">Back</span>
+  <span class="nav-left fl-menu-back focus-outline" data-fl-navigate-back tabindex="1">
```

Comment:
We should only use values `0` and `-1`.

`-1` means not tab-able, and `0` means tab-able. Is there a reason you're using `1` here?

```suggestion
  <span class="nav-left fl-menu-back focus-outline" data-fl-navigate-back tabindex="0">
```

Reply:
Currently, I don't remember why I used tabindex="1" instead of tabindex="0" for menus. I rechecked and it seems to work fine with  tabindex="0", but  I am sure that there was a reason for using tabindex="1".

Reply:
@MaksymShokin You need to find out what the reason is. Otherwise, we should only use `0` (and everywhere else too). See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Reply:
@tonytlwu 
Changed to tabindex="0"